config-handler-router/pom.xml

```diff
@@ -24,6 +24,54 @@
     <packaging>bundle</packaging>
 
     <dependencies>
+        <!-- TODO: Replace all dependencies below this line with shaded jar at version 2.10.0-->
```

Comment:
I believe - correct me if I'm wrong - that @tbatie pushed these temporary dependencies into the root pom so we shouldn't need to dupe them everywhere in admin-console.